### PR TITLE
chore(config): Add FireTV to the validation list for "oldsync" scope.

### PR DESCRIPTION
**Pull request by rfk**
Ref https://bugzilla.mozilla.org/show_bug.cgi?id=1566346

### CODE REVIEW

File: packages/fxa-content-server/server/lib/configuration.js [hunk 1]
```diff
@@ -498,6 +498,7 @@ const conf = (module.exports = convict({
             'https://lockbox.firefox.com/fxa/android-redirect.html',
             'https://accounts.firefox.com/oauth/success/a2270f727f45f648', // Fenix
             'https://accounts.firefox.com/oauth/success/3c49430b43dfba77', // Reference browser
+            'https://accounts.firefox.com/oauth/success/85da77264642d6a1', // Firefox for FireTV
```
Comment (shane-tomlinson) [inline packages/fxa-content-server/server/lib/configuration.js]: Having *this* config here and all other config in cloudops-deployment is a recipe for confusion and disaster! What do we need to do to get this out of here and into cloudops-deployment instead?

**Comment by shane-tomlinson** [comment]
@rfk - should we target this as a point release for train-141 so @mcomella can test?

File: packages/fxa-content-server/server/lib/configuration.js (hunk 1 above)
Comment (shane-tomlinson) [reply packages/fxa-content-server/server/lib/configuration.js]: Opened https://github.com/mozilla/fxa/issues/1903

**Comment by rfk** [comment]
> should we target this as a point release for train-141 

Sure; I assume it will then also need to be forward-ported to 142, and then master. I can wrangle that today.

**Comment by rfk** [comment]
I've changed this to target train-141, but had to pick up https://github.com/mozilla/fxa/pull/1866 in order to get CI (hopefully...) passing.

**Comment by rfk** [comment]
CI now seems to be failing on the language change from https://github.com/mozilla/fxa/pull/1870; I cherry-picked the fix from that PR but it doesn't seem to help, and I don't have any more bandwidth to keep poking at this. @shane-tomlinson how should I unstick it?

**Comment by rfk** [comment]
Woohoo, CI passing! @shane-tomlinson please merge and cut this if it all good to you, otherwise I'll revisit in the morning.

**Pull request merged**